File: java_2_gb/OOP/src/main/java/org/example/seminar5/cw/model/HotDrinks.java
package org.example.seminar4.hw;
import java.math.BigDecimal;

public class HotDrinks extends Drinkables {
    private int temp;
    public HotDrinks(String name, BigDecimal price, Double volume, int temp) {
        super(name, price, volume);
        this.temp = temp;
    }
    public int getTemp() {
        return temp;
    }
    public String toString() {
        return super.toString() + "температура напитка = " + temp;
    }
}

File: java_2_gb/finalTask/Model/Goods/HotDrinks.py
from Model.Goods.Drinkables import Drinkables


class HotDrinks(Drinkables):
    def __init__(self, name, price, volume, temperature):
        super().__init__(name, price, volume)
        self._temperature = temperature

    @property
    def temperature(self):
        return self._temperature

    @temperature.setter
    def temperature(self, new_temperature):
        self._temperature = new_temperature

    def __str__(self) -> str:
        return f"{super().__str__()}, температура: {self.temperature}"